### PR TITLE
Prepare v1.9.2 release

### PR DESCRIPTION
**Pull request by etungsten**
**Issue number:**
N/A

**Description of changes:**
Bumps the version in `Release.toml` and adds change log for v1.9.2

**Terms of contribution:**

By submitting this pull request, I agree that this contribution is dual-licensed under the terms of both the Apache License, version 2.0, and the MIT license.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,13 @@
+# v1.9.2 (2022-08-31)
+
+## Build Changes
+
+* Archive old migrations ([#2357])
+* Update `runc` to version 1.1.4 ([#2380])
+
+[#2357]: https://github.com/bottlerocket-os/bottlerocket/pull/2357
+[#2323]: https://github.com/bottlerocket-os/bottlerocket/pull/2380
```
Comment (etungsten) [inline CHANGELOG.md]:
```suggestion
[#2380]: https://github.com/bottlerocket-os/bottlerocket/pull/2380
```

**Review by kdaula — APPROVED** [review]
LGTM

**Comment by etungsten** [comment]
Push above fixes a PR link

**Review by jpmcb — APPROVED** [review]
🕺🏼

**Review by gthao313 — APPROVED** [review]

**Pull request merged**